requirements/frozen_docs.py2.txt

```diff
@@ -21,7 +21,6 @@ bottle-sqlite==0.1.3
 cached-property==1.5.1
 certifi==2018.10.15
 cffi==1.11.5
-cfme-testcases==0.12
```

Comment:
we still use this, aren't we ?

Reply:
Correct - but its not needed for docs (or at least isn't in the docs template)